### PR TITLE
Create projetinho

### PR DESCRIPTION
**Pull request by SerginhoReis**

### CODE REVIEW

**Comment by Hardisulaeman** [comment]
👍👍👍

**Comment by sofiqrahman** [comment]
That's

**Comment by fuliganakerim** [comment]
Pull

File: projetinho [hunk 1]
```diff
@@ -0,0 +1 @@
+
```
Comment (fuliganakerim) [inline projetinho]: Pull

**Comment by knonomura** [comment]
I close this PR as it is not related to griddb/griddb_nosql repository.

**Pull request closed without merging**